ee/ucp/kubernetes/storage/use-azure-file.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Configuring Azure File Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure File.
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
+Enterprise when running on Microsoft Azure by using Azure File. Operators can
+either pre-provision Azure File Shares to be consumed by Kubernetes Pods or can
+you use the Azure Kubernetes integration to dynamically provision Azure File
+Shares on demand.
+
+## Pre-Requisites
+
+This guide assumes you have already provisioned a UCP environment on to
+Microsoft Azure. The Cluster must be provisioned after meeting all of the
+pre-requisites within [Install UCP on
+Azure](/ee/ucp/admin/install/install-on-azure.md).
+
+Additionally the following guide is using the Kubernetes Command Line tool `$
+kubectl` to provision Kubernetes objects within a UCP cluster. Therefore this
+tool needs to be downloaded, along with a UCP client bundle. For more
+information on configuring CLI access to UCP see [CLI Based
+Access](/ee/ucp/user-access/cli/).
+
+## Manually Provisioning Azure File
+
+An operator can use existing Azure File Shares or manually provision new ones to
+provide persistent storage for Kubernetes Pods. Azure File Shares can be
+manually provisioned in the Azure Portal, using ARM Templates or using the Azure
+Cli. In the below example we have used the Azure Cli to manually provision an
+Azure File Shares. 
+
+### Creating an Azure Storage Account
+
+The first step when manually creating an Azure File Share, is to create an Azure
+Storage Account for the file shares to live in. If you have already provisioned
+a Storage Account, you can move on to [Creating an Azure File
+Share](#creating-an-azure-file-share).
+
+> Note the Azure Kubernetes Driver does not support Azure Storage Accounts
```

Comment:
Both Azure Disks and Azure Files are under the same Azure Storage service; love this callout but do we need it in the Disk storage doc too? Or is it only for File Storage where there is a limitation for Premium?

Reply:
This is only on Azure files that there is a limitation on premium storage. Azure Disk for premium storage works correctly.